src/main/java/com/alkemy/ong/mapper/enums/SlideOperation.java

```diff
@@ -0,0 +1,7 @@
+package com.alkemy.ong.mapper.enums;
+
+public enum SlideOperation {
+
+  ALL, IMAGEORDER;
```

Comment:
MEANINGFUL NAMES: estos nombres no parecen que representaran operaciones. Si miro el proyecto en este momento, para Slides tenemos un LIST y un LIST_IN_ORGANIZATIONS. Creo que esos serian mas representativos.

ON THE OTHER SIDE / BUENAS PRACTICAS: esta no es la mejor implementación, ya que la lista de operaciones en un proyecto real pueden ser montones de combinaciones y haría difícil de setear nombres representativos. Además de la cantidad de código duplicado en el mapper. 
Como comenté en mi anterior revisión, lo ideal sería tener los nombres de los campos de Slide en un enum y pasarle los campos que se desean mapear, de esa forma el código del mapper sería mucho más claro, reutilizable y mantenible.